CodeStarterCamp_Week3/main.swift

```diff
@@ -8,5 +8,44 @@
 
 import Foundation
 
-print("Hello, World!")
+enum Coffee: String {
+    case dark = "Dark Coffee"
+    case latte = "Latte Coffee"
+}
+
+struct Person {
+    var money: Int
+    func purchaseCoffee() {
+        
+    }
```

Comment:
```suggestion
    func purchaseCoffee() {}
```
- 메서드에 기능이 아직 없다면 이렇게 개행 없이 쓰는 것도 좋을 거 같습니다!

Reply:
넵 ! 수정해두겠습니다 솜.